allennlp/tests/nn/util_test.py

```diff
@@ -661,6 +662,7 @@ def test_sequence_cross_entropy_with_logits_averages_batch_correctly(self):
         # Batch has one completely padded row, so divide by 4.
         assert loss.data.numpy() == vector_loss.sum().item() / 4
 
+    @flaky # some time "not almost equal to 7 decimals"
```

Comment:
No need for the comment here; that's already the definition of `flaky`.  I'm not sure why the change you made would make this test flaky, though, while the ones below aren't.

Reply:
It just happened to fail at a very early stage I try to modify the loss. Did not notice if it is related to my changes. It happens once out of 10 rounds. It really looks irrelevant. I just add this to stop the unit test complaining.
My test is using token-average, as in smooth label test. So it is not affected by the unstable problem when using batch-average, in the above flaky test.

However, I also see once test with the token-average complaint "not almost equal to 7 decimals", out of maybe 50+ times local and TC tests. Please refer to [# 7006](http://build.allennlp.org/viewLog.html?buildId=17137) in TC. Need `@flaky` for it also? This may imply all other tests rely on token-average could be flaky. There are 8-9 calls to token-average which is the default. So I will estimate the chance of flaky is ~1/500.

Reply:
Sarcasm is that I get flaky immediately with this build http://build.allennlp.org/viewLog.html?buildId=17245. Alright, I will add `@flaky` to all of those.

Reply:
It seems no problem this part as I discuss in the above conversation. The question here is if you want a minimum `@flaky` flags? If so I remove not necessary ones.

Reply:
I don't see a problem with having a `@flaky` decorator, I was just surprised it's necessary.